backend/app/controllers/users.js

```diff
@@ -205,6 +205,7 @@ module.exports = {
                 roleID: data.roleID,
                 organizationId: data.organizationId,
                 isActive: data.isActive,
+                registered: data.registered,
```

Comment:
So do you check against this on the front end to see if a user was created/inserted? Can't we just check using the userID to know if the User was inserted? If there isn't a userID then we know the insert wasn't successful. And by doing this, I don't think we need to even worry about setting it to true below. The thing is if insert didn't go through then nothing would be returned. 

It's not a bit deal though, but this could probably been a front end only change. :P

Reply:
Eh... we could. But that's a lot of extra work for an otherwise very minor and fleeting thing.

Reply:
Tell you what. Let's accept for now and I'll write a PR to do this later.